vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditModuleLinkController.java

```diff
@@ -18,30 +18,31 @@
 import edu.asu.diging.vspace.core.model.display.IModuleLinkDisplay;
 import edu.asu.diging.vspace.core.services.IModuleLinkManager;
 import edu.asu.diging.vspace.core.services.ISpaceManager;
-
+import edu.asu.diging.vspace.core.exception.ImageDoesNotExistException;
 @Controller
 public class EditModuleLinkController extends EditSpaceLinksController {
 
     @Autowired
     private ISpaceManager spaceManager;
 
     @Autowired
-    private IModuleLinkManager moduleLinkManager;
-
+    private IModuleLinkManager moduleLinkManager;   
+    
     @RequestMapping(value = "/staff/space/link/module/{id}", method = RequestMethod.POST)
     public ResponseEntity<String> editModuleLink(@PathVariable("id") String id, @RequestParam("x") String x,
-            @RequestParam("y") String y, @RequestParam("rotation") String rotation,
-            @RequestParam("moduleLinkLabel") String title, @RequestParam("linkedModule") String linkedModuleId,
-            @RequestParam("moduleLinkLabel") String moduleLinkLabel,
-            @RequestParam("moduleLinkIdValueEdit") String moduleLinkIdValueEdit,
-            @RequestParam("moduleLinkDisplayId") String moduleLinkDisplayId, @RequestParam("type") String displayType,
-            @RequestParam("moduleLinkImage") MultipartFile file) throws NumberFormatException,
-            SpaceDoesNotExistException, LinkDoesNotExistsException, IOException, ImageCouldNotBeStoredException {
+        @RequestParam("y") String y, @RequestParam("rotation") String rotation,
+        @RequestParam("moduleLinkLabel") String title, @RequestParam("linkedModule") String linkedModuleId,
+        @RequestParam("moduleLinkLabel") String moduleLinkLabel,
+        @RequestParam("moduleLinkIdValueEdit") String moduleLinkIdValueEdit,
+        @RequestParam("moduleLinkDisplayId") String moduleLinkDisplayId, @RequestParam("type") String displayType,
+        @RequestParam(value = "moduleLinkImage", required=false) MultipartFile file, @RequestParam(value="editModule-imageId", required=false) String imageId) throws NumberFormatException,
```

Comment:
`editModule-imageId`: it's editing the module link, not the module, right? name should reflect that.